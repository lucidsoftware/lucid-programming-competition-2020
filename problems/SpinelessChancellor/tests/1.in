6
A222
B112
C421
D879
E900
F450
10
A222 yes
B112 yes
C421 yes
D879 yes
E900 yes
F450 yes
A222 yes
B112 yes
C421 yes
D879 no
E900 yes
F450 yes
A222 yes
B112 yes
C421 yes
D879 no
E900 yes
F450 yes
A222 yes
B112 yes
C421 yes
D879 no
E900 yes
F450 yes
A222 yes
B112 yes
C421 yes
D879 no
E900 yes
F450 no
A222 yes
B112 yes
C421 yes
D879 unsure
E900 yes
F450 no
A222 yes
B112 yes
C421 no
D879 unsure
E900 no
F450 no
A222 yes
B112 yes
C421 no
D879 unsure
E900 no
F450 no
A222 yes
B112 yes
C421 no
D879 unsure
E900 no
F450 no
A222 no
B112 no
C421 unsure
D879 unsure
E900 unsure
F450 no
