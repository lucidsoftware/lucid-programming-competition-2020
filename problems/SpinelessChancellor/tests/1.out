A222 yes
B112 yes
C421 abstain
D879 no
E900 abstain
F450 no
